examples/step-67/doc/intro.dox

```diff
@@ -0,0 +1,680 @@
+in time system compactly as
+@f[
+\mathcal M \frac{\partial \mathbf{w}_h}{\partial t} =
+\mathcal L_h(t, \mathbf{w}_h),
+@f]
+or, equivalently,
+@f[
+\frac{\partial \mathbf{w}_h}{\partial t} =
+\mathcal M^{-1} \mathcal L_h(t, \mathbf{w}_h).
+@f]
+
+For hyperbolic systems discretized by high-order discontinuous Galerkin
+methods, explicit time integration of this system is very populator. This is
+due to the fact that the mass matrix $\mathcal M$ is block-diagonal and thus
+easily inverted. In each time step---or stage of a Runge--Kutta scheme---one
+only needs to evaluate the differential operator once using the given data and
+subsequently apply the inverse of the mass matrix. For implicit time stepping,
+on the other hand, one would first have to linearize the equations and then
+iteratively solve the linear system, which involve several residual
+evaluations and at least a dozen of applications of the linearized operator,
+as has been demonstrated in the step-33 tutorial program.
+
+Of course, the simplicity of explicit time stepping comes with a price, namely
+conditional stability due to the so-called Courant--Friedrichs--Lewy (CFL)
+condition. It states that the time step cannot be larger than the fastest
+propagation of information by the discretized differential operator. In more
+modern terms, the speed of propagation corresponds to the largest eigenvalue
+in the discretized operator, and in turn depends on the mesh size, the
+polynomial degree $p$ and the physics of the Euler operator, i.e., the
+eigenvalues of the linearization of $\mathbf F(\mathbf w)$ with respect to
+$\mathbf{w}$. In this program, we set the maximal time step as follows:
+@f[
+\Delta t = \frac{Cr}{p^{1.5}}\left(\frac{1}{\max\left[\frac{\|\mathbf{u}\|}{h_u} +
+           \frac{c}{h_c}\right]}\right).
+@f]
+The power $p^{1.5}$ used for the polynomial scaling is heuristic and
+represents the closest fit for polynomial degrees between 1 and 8, see
+e.g. @cite SchoederKormann2018. In the limit of higher degrees, $p>10$, a
+scaling of $p^2$ is more accurate, related to the inverse estimates typically
+used for interior penalty methods. Regarding the <i>effective</i> mesh sizes
+$h_u$ and $h_c$ used in the formula, we note that the convective transport is
+directional. Thus an appropriate scaling is to use the element length in the
+direction of the velocity $\mathbf u$. The code below derives this scaling
+from the inverse of the Jacobian from the reference to real cell, i.e., we
+approximate $\frac{\|\mathbf{u}\|}{h_u} \approx \|J^{-1} \mathbf
+u\|_{\infty}$. The acoustic waves, instead, are isotropic in character, which
+is why we use the smallest feature size, represented by the smallest singular
+value of $J$, for the acoustic scaling $h_c$. Finally, we need to add the
+convective and acoustic limits, as the Euler equations can transport
+information with speed $\|\mathbf{u}\|+c$.
+
+In this tutorial program, we use a specific variant of explicit Runge--Kutta
```

Comment:
Somewhere here it would probably be good to link to either https://en.wikipedia.org/wiki/Runge%E2%80%93Kutta_methods and/or the book by Hairer and Wanner (and Norsett in the latest edition).